profile/README.md

```diff
@@ -11,6 +11,18 @@ Pangea empowers organizations to ship secure AI applications quickly with the in
   - [Pangea JavaScript tools](https://github.com/pangeacyber/pangea-javascript)
   - [Pangea Python tools](https://github.com/pangeacyber/pangea-python)
 - [Pangea CLI](https://github.com/pangeacyber/pangea-cli)
+- Wrappers
```

Comment:
Nit: maybe `AI provider wrappers` or `Wrappers for AI providers` or `API wrappers`?

Reply:
Went with "Wrappers for AI providers"